ruby_programming/test_driven_development/rspec_introduction_lesson.md

```diff
@@ -1,26 +1,245 @@
 ### Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+
+By now you've had some experience writing actual programs using Ruby.  Presumably, you've been making sure they work by running your script, manually setting up, entering individual inputs, and watching the output for correctness.
+
+Presumably, you've *also* suffered minor concussions from head-desking as you painstakingly repeated this cycle over, and over, and over again on the more complex projects (looking at you, Algorithms and Data Structures).
```

Comment:
These first two paragraphs would fit better in the lesson that comes before this IMO `Why TDD` This is good stuff though so we should definitley just copy and paste this in there.

Reply:
:+1:  is that you @mindovermiles262?

Reply:
Yeah, I'll incorporate these two paragraphs in my intro